src/vsCodeConfig/tasks.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { TaskDefinition, workspace, WorkspaceConfiguration, WorkspaceFolder } from "vscode";
```

Comment:
Why not just the vscode API directly?  Seems like you're only using the commands anyway.

Reply:
I mean it just simplifies my life a little bit. I get the tasks in multiple places and I had to do this each time:
```typescript
const config: WorkspaceConfiguration = workspace.getConfiguration('tasks', folder.uri);
// tslint:disable-next-line: strict-boolean-expressions
const tasks: IFuncTaskDefinition[] = config.get<IFuncTaskDefinition[]>('tasks') || [];
```

Now I can just do this:
```typescript
const tasks: ITask[] = getTasks(folder);
```